public/forms/builder/form.ttl

```diff
@@ -11,96 +11,96 @@
 # form
 ##########################################################
 ext:form a form:Form, form:TopLevelForm ;
-  form:includes ext:propertyGroupL;
+  form:includes ext:sectionL;
   form:includes ext:formNodesL;
   form:includes ext:formListingL;
   form:includes ext:formListingTableL.
 
 ##########################################################
-#  property-group
+#  Section
 ##########################################################
-ext:mainPg a form:PropertyGroup;
+ext:mainPg a form:Section;
     sh:name "Form Builder" ;
     sh:order 0 .
 
-ext:propertyGroupPg a form:PropertyGroup;
+ext:sectionPg a form:Section;
     sh:name "Section" ;
     sh:order 1 .
 
-ext:listingPg a form:PropertyGroup;
+ext:listingPg a form:Section;
     sh:name "Listing" ;
     sh:order 2 .
 
-ext:formFieldPg a form:PropertyGroup;
+ext:formFieldPg a form:Section;
     sh:name "Field" ;
     sh:order 3 .
 
 ##########################################################
-#  PropertyGroup Scope
+#  section Scope
 #  TODO: allow finding triples for a "form:Scope"
 #    with no "sh:path" provided.
-#    We now revert to attaching the propertyGroups to
+#    We now revert to attaching the sections to
 #    the "form:Form" directly.
 #    Which is suboptimal, cause not really in the defined
 #    model. (It's not forbidden though)
 ##########################################################
-ext:propertyGroupS a form:Scope;
-  sh:path sh:group ;
+ext:sectionS a form:Scope;
```

Comment:
Than we have to find something for scope but my SC is good enough? `ext:sectionSC` I would even be happy with `ext:sectionScope`